src/server/scripts/Northrend/FrozenHalls/PitOfSaron/pit_of_saron.cpp

```diff
@@ -129,6 +129,7 @@ class npc_pos_leader : public CreatureScript
                                 n2->GetMotionMaster()->MovePoint(1, NecrolytePos2);
                                 n2->SetUInt32Value(UNIT_NPC_EMOTESTATE, EMOTE_STATE_READY1H);
                             }
+                            // TODO This spell check ins invalid
                             if (SPELL_NECROLYTE_CHANNELING)
```

Comment:
Why not just remove all occurrences of "if (SPELL_NECROLYTE_CHANNELING)"? They are always true.

Reply:
I don't know if this statement is there for a reason. I just added a comment because at that place probably should be a check.

Reply:
Agreed, we should fix this in another PR. I pushed a new commit fixing a typo in the comment, otherwise this PR looks good to me.